symbols/arm9.yml

```diff
@@ -9005,32 +9051,42 @@ arm9:
     - name: DAMAGE_FORMULA_NON_TEAM_MEMBER_MODIFIER
       address:
         NA: 0x20A18E8
+        JP: 0x20A2CBC
       length:
         NA: 0x4
+        JP: 0x4
       description: "The divisor applied to the overall output of the \"shifted base\" damage formula (the sum of the scaled AT, Def, ClampedLn, and DAMAGE_FORMULA_CONSTANT_SHIFT terms) if the attacker is not a team member (and the current fixed room is not the substitute room...for some reason), as a binary fixed-point number with 8 fraction bits (85/64)."
     - name: DAMAGE_FORMULA_LN_PREFACTOR
       address:
         NA: 0x20A18EC
+        JP: 0x20A2CC0
       length:
         NA: 0x4
+        JP: 0x4
       description: "The prefactor to the output of the ClampedLn in the damage formula, as a binary fixed-point number with 8 fraction bits (50)."
-    - name: DAMAGE_FORMULA_AT_PREFACTOR
+    - name: DAMAGE_FORMULA_DEF_PREFACTOR
       address:
         NA: 0x20A18F4
```

Comment:
Good catch, this was actually a mistake. The NA address here should be `0x20A18F0`. I'll update it.